modules/practice/practice.controller.js

```diff
@@ -0,0 +1,19 @@
+//constant imports
+const constants = require("./practice.constant");
+
+const printMessage = (req, res) => {
```

Comment:
This function should be an async function

Reply:
I did not use await keywork so why to make it an async function?

Reply:
OKay!